src/components/FormAccordion.vue

```diff
@@ -70,4 +64,9 @@ export default {
   border: none;
   border-bottom: 1px solid rgba(0, 0, 0, 0.125);
 }
+
+.collapsed > .when-opened,
+:not(.collapsed) > .when-closed {
+  display: none;
+}
```

Comment:
This was taken straight form the docs at https://bootstrap-vue.js.org/docs/components/collapse/#accordion-support.